CODEX_IMPLEMENTATION_PLAN.md

```
@@ -0,0 +1,1460 @@
+  jest.clearAllMocks();
+
+  const { importModule } = require('../../src/esm');
+  importModule.mockResolvedValue({
+    Codex: MockCodex,
+  });
+});
+```
+
+#### 2.3 Key Test Cases
+
+**Test: Basic successful call**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)